### ISSUE TITLE
Discord crashes browser. Engine-agnostic.

### ISSUE THREAD

**Issue by RomanHargrave** [crmarsh/discord-linux-bugs#94]
This is me "moving" my previous issue.

Discord has managed to crash:

* Firefox (completely)
* Google Chrome (tab process)
* Embedded Webkit (discord client)

I have no debugging info at this time.

**Comment by crmarsh** [comment]
The thing these all have in common is WebRTC (I think?) so probably you are hitting a bug in there. I found one, for instance, with reloading the PulseAudio daemon while in a voice channel. Is there something going on at the time of the crash to help narrow this down? Is there something in that console log I should be noticing?

**Comment by RomanHargrave** [comment]
No, the information from the console is totally inconsequential, which is frustrating, and I have yet to find any pulse-crash correlation, either. It's really strange.

**Comment by FreeSlave** [comment]
For me it crashes Chromium completely, then opens Chromium again with links to images I viewed in Discord before.
